prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -0,0 +1,515 @@
+import os
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from zone_profile_generator import zone_shp_overlay
+
+
+def get_zone_floor_area(iso_list):
+    """Computes the zone floor area for each ISO.
+
+    :param list of str iso_list: list of abbrev. names of ISO.
```

Comment:
I have changed  the code to take in already defined ISO from a for loop, this would ensure only the valid string of ISO is taken into the plotting function.